examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<blockquote>To model TID in PASEOS, we can use custom properties. First of all, we define the a property called <code>total_ionizing_dose</code>. We then create a dedicated an update function <code>prop_update_fn</code> to compute the dose over time.</blockquote><p>To model TID in PASEOS, we can use custom properties. First of all, we define a property called <code>total_ionizing_dose</code>. We then create a dedicated update function <code>prop_update_fn</code> to compute the TID over time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>